docs/user-guide/manage-environments.md

```diff
@@ -178,7 +178,27 @@ To inspect the deployed application in KubeRocketCI portal, take the following s
 
     ![Inspect application](../assets/user-guide/environments/application_terminal.png "Inspect application")
 
+### Delete Application
+
+There are two buttons in the environment which can be used in the portal:
+
+  ![Clean button](../assets/operator-guide/edp-portal-clean-application.png "Clean button")
+
+Although these buttons appear similar, they serve different purposes. To not mix up these two buttons, let's review both of them:
+
+  * **Delete** - Deletes Argo CD application set and associated resources within the environment. Additionally, this button allows to delete environment applications one by one.
+  * **Clean** - Executes a delete pipeline with custom logic. This button comes in handy when your environment has a custom deploy pipeline.
```

Comment:
"This button is useful when you need to perform custom cleanup actions, e.g. delete cloud resources, database, rollback transactions, etc."